lib/config_loader.rb

```diff
@@ -0,0 +1,13 @@
+class ConfigLoader
+  def initialize(config_file)
+    @config =  YAML.load(File.open(config_file, 'r'))
+  end
+  def account_sid
+    fail "Account SID Missing" unless @config.key?('Account SID')
+    @config['Account SID']
+  end
+  def auth_token
+    fail "Auth Token Missing" unless  @config.key?('Auth Token')
+    @config['Auth Token']
+  end
+end
```

Comment:
This wasn't really needed, but quite clever. Avoid 'er' at the end of class names - if it is a service object it would end in a verb.